x-pack/plugins/apm/server/lib/transaction_groups/correlations/get_significant_terms_agg.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ESFilter } from '../../../../typings/elasticsearch';
+
+export function getSignificantTermsAgg(
```

Comment:
Maybe you could write a unit test for both `getSignificantTermsAgg` and `formatAggregationResponse`?